modules/configuration/php/project.class.inc

```diff
@@ -53,7 +53,7 @@ class Project extends \NDB_Form
         $projectList    = \Utility::getProjectList();
         $subprojectList = \Utility::getSubprojectList();
         $projects       = array();
-        foreach ($projectList as $ProjectID => $Name) {
+        foreach ($projectList as $ProjectID) {
```

Comment:
I don't think these are equivalent, after your change the value is actually being inserted in the `$ProjectID` variable, not the key, before your change the key was assigned to that variable